src/das.ts

```
@@ -16,6 +18,21 @@ import {
 
+function validateDisplayOptions(displayOptions?: DisplayOptions) {
+  if (!displayOptions) return;
+  
+  const allowedOptions = ['showCollectionMetadata'];
+  const providedOptions = Object.keys(displayOptions);
+  
+  const invalidOptions = providedOptions.filter(opt => !allowedOptions.includes(opt));
+  if (invalidOptions.length > 0) {
+    throw new Error(
+      `The following display options are not supported with MPL Core: ${invalidOptions.join(', ')}. ` +
+      'Only showCollectionMetadata is supported.'
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)